beacon_node/beacon_chain/src/attestation_rewards.rs

```diff
@@ -134,10 +133,8 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         let spec = &self.spec;
 
         // Calculate ideal_rewards
-        let participation_cache = ParticipationCache::new(&state, spec)?;
-        process_justification_and_finalization(&state, &participation_cache)?
-            .apply_changes_to_state(&mut state);
-        process_inactivity_updates(&mut state, &participation_cache, spec)?;
+        process_justification_and_finalization(&state)?.apply_changes_to_state(&mut state);
+        process_inactivity_updates_slow(&mut state, spec)?;
```

Comment:
Two potential issues here around `process_inactivity_updates_slow()`:
1) This function calls `process_epoch_single_pass()` which requires the committee caches be built before hand.
2) The comment on `process_inactivity_updates_slow()` indicates that it's supposed to only be used in testing. By utilizing it here, we are relying on it to provide the data for the `/eth/v1/beacon/rewards/attestations/{epoch}` `beacon-api` endpoint

Reply:
1. added calls to build the committee caches in `process_epoch_single_pass`
2. clarified the comment that we're OK to use `_slow`, just not in mainline block/epoch processing